plugins/alias-finder/alias-finder.plugin.zsh

```diff
@@ -0,0 +1,46 @@
+alias-finder() {
+  local cmd=""
+  local exact=""
+  local longer=""
+  for i in $@; do
+    case $i in
+      -e|--exact) local exact=true;;
+      -l|--longer) local longer=true;;
+      *) 
+        if [[ -z $cmd ]]; then
+          local cmd=$i
+        else
+          local cmd="$cmd $i"
+        fi
+        ;;
+    esac
+  done
+  cmd=$(sed 's/[].\|$(){}?+*^[]/\\&/g' <<< $cmd)
+  if [[ $(wc -l <<< $cmd) == 1 ]]; then
+    while [[ $cmd != "" ]]; do
+      if [[ $longer = true ]]; then
+        local wordStart="'{0,1}"
+      else
+        local wordEnd="$"
+        local multiWordEnd="'$"
+      fi
+      if echo $cmd | grep '^[^ ]*$' > /dev/null; then
```

Comment:
What is this line supposed to do?

Reply:
It checks if `cmd` is a single word (i.e. has no spaces). I have simplified it at https://github.com/robbyrussell/oh-my-zsh/pull/7768/commits/58f7dfd4cb204d644efbdb9135af5d61f71c3c91